students/jeff_shabani/session06/test_mailroom_4.py

```diff
@@ -57,6 +82,23 @@ def test_membership_in_dictionary(self):
 
         self.assertIn('Bobby', mailroom_4.create_new_donors_dict())
 
+    def test_get_value_pass(self):
+        """
+        test that get_value function returns input value
+        if input value is the correct type
+        :return: int
+        """
+        self.assertEqual(get_value(45, int), 45)
+
+    @unittest.expectedFailure
+    def test_get_value_fail(self):
+        """
+        test that get_value function returns a message to
+        user if input value is incorrect type
+        :return:
+        """
+        self.assertEqual(get_value('amoimt', int), 54)
```

Comment:
So your testing copies of the functions rather than the actual functions in mailroom.py.  As a result these tests aren't really useful, as if you chaged the `get_value` function in mailroom.py these tests would still pass.